src/App.tsx

```diff
@@ -1,14 +1,70 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+
+import { Todo } from './types/Todo';
+
+const TODOS_DEFAULT: Todo[] = [];
+
+const TODO_DEFAULT: Todo = {
+  id: 0,
+  title: '',
+  completed: false,
+  userId: 0,
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState(TODOS_DEFAULT);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [todoId, setTodoId] = useState(0);
+  const [todo, setTodo] = useState(TODO_DEFAULT);
+  const [statusFilter, setStatusFilter] = useState('all');
+  const [titleFilter, setTitleFilter] = useState('');
+
+  useEffect(() => {
+    getTodos().then(result => {
+      setTodos(result);
+      setVisibleTodos(result);
```

Comment:
Hello. VisibleTodos - it's the final array after filtering todosFfomServer, which i pass to the TodoList component to render. But I should pass something to the TodoList to avoid empty list after page loading. So as result I have todosFromServer as the initial array and visibleTodos - as the modified array to show, which is similar to initial array after page loaded.